apps/hermes/server/src/api/types.rs

```diff
@@ -326,17 +326,70 @@ pub struct PriceFeedMetadata {
 }
 
 #[derive(Debug, Serialize, Deserialize, PartialEq, ToSchema)]
-#[serde(rename_all = "lowercase")]
+#[serde(rename_all = "snake_case")]
 pub enum AssetType {
     Crypto,
     FX,
     Equity,
-    Metals,
+    Metal,
     Rates,
+    CryptoRedemptionRate,
 }
 
 impl Display for AssetType {
     fn fmt(&self, f: &mut Formatter) -> FmtResult {
-        write!(f, "{:?}", self)
+        match self {
+            AssetType::Crypto => write!(f, "crypto"),
+            AssetType::FX => write!(f, "fx"),
+            AssetType::Equity => write!(f, "equity"),
+            AssetType::Metal => write!(f, "metal"),
+            AssetType::Rates => write!(f, "rates"),
+            AssetType::CryptoRedemptionRate => write!(f, "crypto_redemption_rate"),
+        }
+    }
+}
```

Comment:
Maybe implement it using serde? It's an extra allocation but that shouldn't matter too much.
```
impl Display for AssetType {
    fn fmt(&self, f: &mut Formatter) -> FmtResult {
        let value = serde_json::to_value(self).unwrap();
        write!(f, "{}", value.as_str().unwrap())
    }
}
```
(You can also use serde_value instead of serde_json).